manufacturing-client/src/main.rs

```diff
@@ -401,16 +417,19 @@ async fn main() -> Result<()> {
             // and it might request MACAddress as the mfg_string_type. What it cannot do
             // is select the iface for the client, so we must set it ahead of time.
             // Thereby, we are just getting this value if provided, hiding errors.
```

Comment:
here if you can delete the `// Thereby .... ` part and write that we are going to try to get a default iface if possible